### PR TITLE
Potentially fix dependency PRs not having lockfiles

### PR DESCRIPTION
**Pull request by AshCorr**
## What does this change?

Removes the configuration for the DCR subfolder from dependabot. Dependabot seems to have trouble understanding yarn workspaces if it doesn't have access to the root yarn package files (such as the case in the DCR config). See https://github.com/dependabot/feedback/issues/740

I'm hoping this will allow Dependabot to keep updating DCR dependencies and still play nicely with AR, but I'm not entirely sure it will. Theres no way to tell Dependabot to ignore the AR folder when updating our root yarn workspace.

Also adds AWS libs to the list of ignored dependencies for DCR as we're using CDK in DCR too now.

## Open prs at time of writting with no lockfiles

https://github.com/guardian/dotcom-rendering/pull/7568
https://github.com/guardian/dotcom-rendering/pull/7535
https://github.com/guardian/dotcom-rendering/pull/7424
https://github.com/guardian/dotcom-rendering/pull/7377
https://github.com/guardian/dotcom-rendering/pull/5649
https://github.com/guardian/dotcom-rendering/pull/5167 @mxdvl

Basically any PR with the "DCR Dependency" label.

### CODE REVIEW

**Review by JamieB-gu — APPROVED** [review]

**Review by mxdvl — APPROVED** [review]
🤞

**Comment by AshCorr** [comment]
Lets see if this works!
